test-failure-policy

```diff
@@ -66,6 +68,12 @@ def main():
             print("due to failure of test harness or framework")
             return 1
 
+        if not number and opts.all:
+            number, img = check_all_known_issues(api, output, opts.image)
+            if number:
+                print(f"Known issue #{number} in {img}")
+                return 77
```

Comment:
I wonder if this is actually a good idea. IMHO the test should still fail, and only give you the message that the naughty is already known. Like that the test run would succeed, nobody would look at the output, and then we'd just land the image and have broken main.

This might need quite some restructuring, though.

Reply:
Right! We can return something else, the `run-test` code is:

```
                  if proc.returncode == 77:
                      self._print_test(skip_reason="# SKIP {0}".format(reason.decode("utf-8")))
                      return None, 0
```

So I think we can "abuse" retry reason? Or else we need to extend `_print_test` to print something arbitrary.

Reply:
Honestly a new status would be nice, so we can parse it nicely in `log.html`, so maybe `"# KNOWN ISSUE {0}"`

Reply:
We can also abuse? `self.output`, need to think how we test this in CI.